.gitignore

```diff
@@ -13,3 +13,4 @@ lerna-debug.log
 *.iml
 .idea/
 .vscode
+jsconfig.json
```

Comment:
Hmm?

Reply:
vs code configuration file, didn't want to commit it.

Reply:
They've already been committed, didn't want to push any more changes?

Reply:
right it has personal setup in it now.